artifacts/artifact.py

```diff
@@ -86,3 +86,40 @@ def AppendSource(self, type_indicator, attributes):
 
     self.sources.append(source_object)
     return source_object
+
+  def AsDict(self):
+    """Converts artifact to dict from ArtifactDefinition
+
+    Returns:
+      dict representation of ArtifactDefinition instance
+    """
+    sources = []
+    for source in self.sources:
+      source_definition = {
+          'type': source.type_indicator,
```

Comment:
Make sure all your text string are unicode safe either with u'' or with importing the right future